src/web/client/WebExtensionContext.ts

```diff
@@ -645,6 +657,47 @@ class WebExtensionContext implements IWebExtensionContext {
     public getVscodeWorkspaceState(key: string): IEntityInfo | undefined {
         return this._vscodeWorkspaceState.get(key);
     }
+
+    // Graph client
+    public async graphClientAuthentication(firstTimeAuth = false) {
+        const accessToken = await graphClientAuthentication(firstTimeAuth);
+        this.setGraphToken(accessToken);
+    }
+
+    public setGraphToken(graphToken: string) {
+        this._graphToken = graphToken;
```

Comment:
no need for this function in any case - we just update the private variable as this._variable = value